docs/api-guide/fields.md

```diff
@@ -68,6 +68,15 @@ When serializing the instance, default will be used if the object attribute or d
 
 Note that setting a `default` value implies that the field is not required. Including both the `default` and `required` keyword arguments is invalid and will raise an error.
 
+Notes Regarding default value propagation from model to serializer:
+
+all the default values of model will be pass as default to serializer and to the options method.
```

Comment:
```suggestion
all the default values of model will pass as default to the serializer and to the options method.
```

Reply:
I think that this would be more correct from `English`, I am not native anyohow...
```
- all the default values of the model will be passed as a `default` to the serializer and to the `OPTIONS` method.
```